not-paid-blur.js

```diff
@@ -0,0 +1,27 @@
+// Next time use letsdeel.com to make sure you get paid
+// author Luca Fabbri <luca@zepfiro.com>
+
+(function(){
+	/* change these variables as you wish */
+	var due_date = new Date('2017-02-27');
+    var days_deadline = 60;
```

Comment:
please use const on these variables and fix the identation

Reply:
Disagree. There’s no build system in place an es6 features are discouraged due to browser compatibility. 

See #10 for further discussion.